src/platforms/affinity/index.ts

```diff
@@ -0,0 +1,17 @@
+import { client } from '@/platforms/affinity/client';
+import { icon } from '@/platforms/affinity/icon';
+import { auth, platform } from '@/sdk';
+
+export default platform('affinity', {
+  auth: auth.basic({
+    questions: [{ type: 'text', id: 'password', label: 'API Token' }],
+  }),
```

Comment:
should the label for this be API token?

Reply:
nope. The funny thing you'll start to see is that there are quite a few integrations that uses api token in basic auth (username:password + base64 encode

Reply:
Oh I meant the `label` attribute on line 7 where it says `'API Token'`, makes sense with what you said